cluster-scope/base/clusterloggings/clusterLogging.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
```

Comment:
I like this much more :+1: We might want to move it to a separate app (outside of the `cluster-scope`) if it's deployable without a cluster-admin permissions. I need to test this out.